Formula/kde-kplotting.rb

```diff
@@ -0,0 +1,64 @@
+class KdeKplotting < Formula
+  desc "Lightweight plotting framework"
+  homepage "https://api.kde.org/frameworks/kplotting/html/index.html"
+  url "https://download.kde.org/stable/frameworks/5.79/kplotting-5.79.0.tar.xz"
+  sha256 "c67238fb1395197b2ee2a65f327458132106df6110765952bd979b11cfbb5c51"
+  license all_of: [
+    "GPL-2.0-or-later",
+    "LGPL-2.0-or-later",
+  ]
+  head "https://invent.kde.org/frameworks/kplotting.git"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "doxygen" => :build
+  depends_on "graphviz" => :build
+  depends_on "kde-extra-cmake-modules" => [:build, :test]
+  depends_on "ninja" => :build
+
+  depends_on "qt@5"
+
+  def install
+    args = std_cmake_args
+    args << "-D BUILD_QCH=ON"
+    args << "-D BUILD_TESTING=OFF"
+    args << "-D BUILD_TESTS=OFF"
+    args << "-D BUILD_UNITTESTS=OFF"
+    args << "-D CMAKE_INSTALL_BUNDLEDIR=#{bin}"
+    args << "-D KDE_INSTALL_LIBDIR=lib"
+    args << "-D KDE_INSTALL_PLUGINDIR=lib/qt5/plugins"
+    args << "-D KDE_INSTALL_QMLDIR=lib/qt5/qml"
+    args << "-D KDE_INSTALL_QTPLUGINDIR=lib/qt5/plugins"
```

Comment:
Build a single array instead of adding to it one-by-one. Given this is passed directly to `cmake`: it's not clear why an array is needed.